modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/mqtt2opcua/BuiltinJsonSchema.java

```diff
@@ -52,8 +52,10 @@ public BuiltinJsonSchema() {
             classToJsonSchema.put(BuiltinDataType.Boolean,
                     createJsonSchemaForBuiltinType("Boolean JsonSchema", BuiltinDataType.Boolean));
 
-            classToJsonSchema.put(BuiltinDataType.SByte, createJsonSchemaForBuiltinType(" SByte JsonSchema", BuiltinDataType.SByte));
-            classToJsonSchema.put(BuiltinDataType.Byte, createJsonSchemaForBuiltinType(" Byte JsonSchema", BuiltinDataType.Byte));
+            classToJsonSchema.put(BuiltinDataType.SByte,
+                    createJsonSchemaForBuiltinType(" SByte JsonSchema", BuiltinDataType.SByte));
+            classToJsonSchema.put(BuiltinDataType.Byte,
+                    createJsonSchemaForBuiltinType(" Byte JsonSchema", BuiltinDataType.Byte));
```

Comment:
Is the leading space necessary or by mistake?

Reply:
i think it is a mistake